src/modelinstance.cpp

```diff
@@ -140,6 +117,35 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
         SPDLOG_DEBUG("model: {}, version: {}; reshaping inputs is not required", getName(), getVersion());
     }
 
+    for (const auto& pair : networkInputs) {
+        const auto& name = pair.first;
+        auto input = pair.second;
+
+        // Data from network
+        auto precision = input->getPrecision();
+        auto layout = input->getLayout();
+        auto shape = input->getTensorDesc().getDims();
+
+        // Data from config
+        if (config.getLayout().size()) {
```

Comment:
Is it possible to get rid of layout dead code in this MR?

Reply:
Can we do it in separate PR? Here I'd like to have just clean fix for the issue with wrong input details reading order. 
Refactor with layout param removal could be done separately don't you think?

Reply:
Right